playbooks/adhoc/uninstall.yml

```diff
@@ -26,6 +26,20 @@
 - hosts: nodes
   become: yes
   tasks:
+  - name: Remove dnsmasq dispatcher
+    file:
+      path: "{{ item }}"
+      state: absent
+    with_items:
+      - /etc/dnsmasq.d/origin-dns.conf
+      - /etc/dnsmasq.d/origin-upstream-dns.conf
+      - /etc/dnsmasq.d/openshift-ansible.conf
+      - /etc/NetworkManager/dispatcher.d/99-origin-dns.sh
+    when: openshift_use_dnsmasq | default(true) | bool
+  - service:
+      name: NetworkManager
+      state: restarted
+    when: openshift_use_dnsmasq | default(true) | bool
```

Comment:
I want to do this immediately as other tasks will start to fail if it's not done here.